src/classes/User.js

```diff
@@ -0,0 +1,41 @@
+class User {
+  constructor(userData = {}) {
+    this.id = userData.id;
+    this.name = userData.name;
+    this.pantry = userData.pantry;
+    this.recipesToCook = [];
+  }
+
+  addRecipesToCook(recipe) {
+    this.recipesToCook.push(recipe);
+  }
+  
+  removeRecipesToCook(recipe) {
+    let recipeToRemove = this.recipesToCook.indexOf(recipe);
+    this.recipesToCook.splice(recipeToRemove, 1);
+    return this.recipesToCook;
+  }
+  filterRecipeByTag(tag) {
+    let filteredRecipesByTag = this.recipesToCook.filter((recipe) => {
+      if (recipe.tags.includes(tag)) {
```

Comment:
You could rewrite lines 20-21 as 
`return recipe.tags.includes(tag)`, because .includes() returns a boolean as its result.

Not saying you should, just that you could! The way you wrote it is superior in terms of developer empathy because it's more obvious what it's doing.

Reply:
Thanks for the suggestion. We will keep with the developer empathy and maintain the extra line of code.